default.nix

```diff
@@ -7,14 +7,18 @@ let
   f = { mkDerivation, stdenv
       , mtl
       , syb
-      , base
       , text
+      , array
+      , bytestring
+      , base
+      , lens
```

Comment:
Do you have Awk? If so,
```
!awk '{ print length, $0 }' | sort -n | cut -d' ' -f2-
```
from visual mode.

Reply:
(otherwise, sort them by length, but keep mkDerivation and stdenv at the top)

Reply:
I dunno, I'd prefer if they were alphabetical.

Reply:
They're sorted by length in the cabal file :< I also sort my imports by length, though in reverse order.